data/json/mapgen/toxic_dump.json

```diff
@@ -17,56 +17,56 @@
     "om_terrain": [ "toxic_dump" ],
     "weight": 100,
     "object": {
-      "fill_ter": "t_dirt",
+      "fill_ter": "t_thconc_floor",
       "rows": [
-        "        x    xxx        ",
-        "       xxxxxxxxxxxxx    ",
```

Comment:
I think there being more barren vegetation here is intended due to the radiation and other toxic stuff. I'd maybe add an increased chance to spawn dead or barren vegetation around the fence.